styles.css

```diff
@@ -82,3 +82,109 @@ ul {
   align-items: center;
   margin: auto;
 }
+
+/* css for the second part of the project */
+
+.works {
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- [ ] Please make sure to add the margins (top, bottom, left, right) that were defined in the design in your works section :+1: